### PR TITLE
lection_4_HW

### PR DESCRIPTION
**Pull request by VyacheslavReetz**
LECTION_4
3 Files

### CODE REVIEW

File: lection4/task3.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+let age_1 = prompt('Hi there! Enter your age below');
```
Comment (anatoly-karpovich) [inline lection4/task3.js:3]: Please provide related html file

File: lection4/task2.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+let age_1 = '12';
+const age_2 = 18;
+const age_3 = 60;
+
+if (typeof age_1 === 'string') {
+    age_1 = +age_1;
```
Comment (anatoly-karpovich) [inline lection4/task2.js:8]: No checks for NaN values. Please add required checks

File: lection4/task3.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+switch (true) {
+  case age_1 < age_2 && age_1 > 0: {
+    alert('You don’t have access cause your age is ' + age_1 + '. It’s less then ' + age_2);
+    break;
+  }
+  case age_1 >= age_2 && age_1 < age_3: {
+    alert('Welcome!');
+    break;
+  }
+  case age_1 >= age_3 && age_1 <= 127: {
```
Comment (anatoly-karpovich) [inline lection4/task3.js]: There were no requirements to add boundary value 127 years) no need to add not required staff)

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

File: lection4/task3.js (hunk 3 above)
Comment (VyacheslavReetz) [reply lection4/task3.js]: but, its a limit of people life for our days(

File: lection4/task2.js (hunk 2 above)
Comment (VyacheslavReetz) [reply lection4/task2.js:8]: for what?
just for not run line 8?)
if age_1 =  NaN -> Technical works

File: lection4/task3.js (hunk 1 above)
Comment (VyacheslavReetz) [reply lection4/task3.js:3]: @anatoly-karpovich done

**Review by anatoly-karpovich — APPROVED** [review]
Good job!

**Pull request merged**